st2actions/st2actions/config.py

```diff
@@ -52,14 +52,11 @@
 ]
 CONF.register_opts(ssh_runner_opts, group='ssh_runner')
 
-workflow_opts = [
-    cfg.StrOpt('url', default='http://localhost:8989', help='Mistral API server root endpoint.')
-]
-CONF.register_opts(workflow_opts, group='workflow')
-
 mistral_opts = [
-    cfg.StrOpt('v2_base_url', default='http://localhost:8989/v2/',
+    cfg.StrOpt('v2_base_url', default='http://localhost:8989/v2',
                help='Mistral v2 API server root endpoint.'),
+    cfg.IntOpt('max_attempts', default=180),
```

Comment:
Hm, 180 attempts ((180-1)*5)?

This seems way too much to me - or is there a good reason for such a high number of attempts? If we do need such a higher number of attempts / retry for so long we should maybe instead try with exponentiation retry delay which is reset on success.